x-pack/plugins/security_solution/public/common/lib/cell_actions/helpers.tsx

```diff
@@ -35,11 +34,6 @@ export const COLUMNS_WITH_LINKS = [
     fieldType: IP_FIELD_TYPE,
     label: i18n.EXPAND_IP_DETAILS,
   },
-  {
-    columnId: SIGNAL_RULE_NAME_FIELD_NAME,
-    label: i18n.VIEW_RULE_DETAILS,
-    linkField: 'kibana.alert.rule.uuid',
-  },
```

Comment:
I think it won't make UX worse if we will have a link for a rule name to redirect to the rule page, and keep the consistency between two views - grid and event rendered view.